pkg/controller/jobframework/reconciler.go

```diff
@@ -601,3 +602,14 @@ func (r *genericReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	}
 	return builder.Complete(r)
 }
+
+// resets the minCount for all podSets if the PartialAdmission feature is not enabled
+func restMinCounts(in []kueue.PodSet) []kueue.PodSet {
```

Comment:
```suggestion
func resetMinCountsIfDisabled(in []kueue.PodSet) []kueue.PodSet {
```

Reply:
the full version wold be too long , I'd need to specify what is disabled as well

Reply:
then just `resetMinCounts`

Reply:
done